### DISCUSSION TITLE
Frigate+ API not working

### DISCUSSION THREAD

**Discussion by toadee** [blakeblackshear/frigate#9672] [Ask A Question] [answered]
I have the environment variable set but for some reason there is still no upload indicator on my events.

Is there something else that I am perhaps missing to enable it.

Currently running version 0.13.1-34fb1c2

**Comment by NickM-27** [comment, marked as answer]
do you have snapshots enabled? that is required

**Reply by toadee** [reply]
I did not. Enabled and the button is showing.
